.prettierrc

```diff
@@ -4,6 +4,7 @@
   "useTabs": false,
   "semi": true,
   "singleQuote": true,
+  "jsxSingleQuote": true,
```

Comment:
If you add formatting rule, please format all other components, or undo this change